src/Microsoft.Identity.Client/MsalErrorMessage.cs

```diff
@@ -262,5 +262,7 @@ public static string CustomWebUiRedirectUriMismatch(string expectedUri, string a
 
         public const string B2CAuthorityHostMisMatch = "The B2C authority host that was used when creating the client application is not the same authority host used in the AcquireToken call. " +
            "See https://aka.ms/msal-net-b2c for details. ";
+
+        public const string FunctionalityNotAvailableInTokenCacheCallback = "This functionality is not available inside of a token cache callback.";
```

Comment:
As a dev, might be useful to have a little more context here and maybe a mitigation step or what to do now/next?

Reply:
+1: #Supportability, aka.ms with more info, if not possible to add here. A developer should be able to self help based on the error message

---
In reply to: [278222563](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/1088#discussion_r278222563) [](ancestors = 278222563)

Reply:
changed the docs since it's specifically the setter methods for the callbacks that can't be called within a callback.  happy to work on wording, don't think this needs a full aka.ms link.